codalab/lib/upload_manager.py

```diff
@@ -470,8 +472,6 @@ def upload_to_bundle_store(
                     params={'success': False, 'error_msg': f'Bypass server upload error. {err}',},
                 )
                 raise err
-            else:
```

Comment:
why remove this?

Reply:
You can't see it in this abridged view, but if you look at the full file the code looks ilke: 
```
 try:
                ....
                self._client.update_bundle_state(bundle['id'], params={'success': True})
            except Exception as err:
                ...
            else:
                self._client.update_bundle_state(bundle['id'], params={'success': True})
```
What does this mean? Well, the 'else' branch triggers in a try-catch block when no error is thrown. In other words, when no errors was thrown, the ` self._client.update_bundle_state(bundle['id'], params={'success': True})` call was triggered twice. This wasn't an issue (except for efficiency reasons) before since the update wasn't incremental (it just reset the whole value), but it was for me since the disk increment happened twice.